include/cantera/kinetics/RxnRates.h

```diff
@@ -451,6 +451,69 @@ class ChebyshevRate
     vector_fp dotProd_; //!< dot product of chebCoeffs with the reduced pressure polynomial
 };
 
+//! Langmuir-Hinshelwood reaction rate type
+class LangmuirRate
+{
+public:
+    //! return the rate coefficient type.
+    static int type() {
+        return LANGMUIR_REACTION_RATECOEFF_TYPE;
+    }
+     //! Default constructor.
+    LangmuirRate();
+     /// Constructor.
+    /// @param A pre-exponential. The unit system is
+    ///     (kmol, m, s). The actual units depend on the reaction
+    ///     order and the dimensionality (surface or bulk).
+    /// @param b Temperature exponent. Non-dimensional.
+    /// @param E Activation energy in temperature units. Kelvin.
```

Comment:
While both options are valid, I would suggest using `//!` to mark Doxygen comment blocks rather than `///`, given that the former is far more widely used within Cantera.